560/step2_defaultdict.py

```diff
@@ -0,0 +1,17 @@
+from collections import defaultdict
+from typing import List
+
+
+class Solution:
+    def subarraySum(self, nums: List[int], k: int) -> int:
+        prefix_sum_to_count = defaultdict(int)
+        prefix_sum_to_count[0] = 1
```

Comment:
ご参考までにdefaultdictの初期化は以下のように一行でも書くことができますね。
ただ横に長くなり読みづらくなる場合もあるので、今のままでもよいかと思いました。

```suggestion
        prefix_sum_to_count = defaultdict(int, {0: 1})
```

Reply:
defaultdictの初期化時に初期値の設定も同時にできるのですね、把握していなかったので勉強になりました！簡潔に書けて良いですね。